Gems/LyShine/Code/Source/World/UiCanvasOnMeshComponent.cpp

```diff
@@ -165,9 +166,9 @@ bool UiCanvasOnMeshComponent::ProcessHitInputEvent(
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 void UiCanvasOnMeshComponent::OnCanvasLoadedIntoEntity(AZ::EntityId uiCanvasEntity)
 {
-    if (uiCanvasEntity.IsValid() && !m_renderTargetOverride.empty())
+    if (uiCanvasEntity.IsValid() && m_attachmentImageAssetOverride)
     {
-        EBUS_EVENT_ID(uiCanvasEntity, UiCanvasBus, SetRenderTargetName, m_renderTargetOverride);
+        EBUS_EVENT_ID(uiCanvasEntity, UiCanvasBus, SetAttachmentImageAsset, m_attachmentImageAssetOverride);
```

Comment:
nit: It's recommended to avoid ebus macros in preference to full fledged ebus calls like
AZ::UiCanvasBus::Event(uiCanvasEntity, &UiCanvasBus::Events::SetAttachmentImageAsset, m_attachmentImageAssetOverride);